agent/util-scripts/unittests

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-export LANG=C
-export LC_ALL=C
+export LANG=C.UTF-8
+export LC_ALL=C.UTF-8
```

Comment:
See #1863.  That PR shows that this change does not affect the existing code either, and so we can merge that ahead of this change to avoid inconsistencies across the existing test frameworks.

Reply:
fixed